infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/BaseSnapshotReader.java

```diff
@@ -280,9 +281,9 @@ public void reset(long ts) {
      * Used to bookkeeping the stream information for the current processing stream
      */
     public static class OpaqueStreamIterator {
-        private String name;
```

Comment:
these members need not be made default AFAI understand.  Only the OpaqueStreamIterator class is used in LogicalGroupSnapshotReader

Reply:
actually it's needed when accessing currentStreamInfo, like 
```
        if (isReadingStreamToClear || !currentStreamHasNext()) {
            log.debug("Snapshot log reader finished reading stream id={}, name={}", currentStreamInfo.uuid,
                    currentStreamInfo.name);
            currentStreamInfo = null;
```